docs/features/recent notes.md

```diff
@@ -7,10 +7,18 @@ Quartz can generate a list of recent notes based on some filtering and sorting c
 
 ## Customization
 
-- Changing the title from "Recent notes": pass in an additional parameter to `Component.RecentNotes({ title: "Recent writing" })`
```

Comment:
can we keep this as a bullet list to make it consistent with other documentation pages?

Reply:
can totally do, do you prefer me to keep the bullet list on this one, or should I make a table everywhere else?

Reply:
let's keep the bullet list for now :)